src/Hedgehog/Property.fs

```diff
@@ -375,87 +356,55 @@ module Property =
     let private reportWith (renderRecheck : bool) (size : Size) (seed : Seed) (p : Property<unit>) : Report =
         reportWith' renderRecheck size seed 100<tests> p
 
-#if !FABLE_COMPILER
-    [<CompiledName("Report")>]
-#endif
     let report' (n : int<tests>) (p : Property<unit>) : Report =
         let seed = Seed.random ()
         reportWith' true 1 seed n p
 
-    [<CompiledName("Report")>]
     let report (p : Property<unit>) : Report =
         report' 100<tests> p
 
-#if !FABLE_COMPILER
-    [<CompiledName("Check")>]
-#endif
     let check' (n : int<tests>) (p : Property<unit>) : unit =
         report' n p
         |> Report.tryRaise
 
-    [<CompiledName("Check")>]
     let check (p : Property<unit>) : unit =
         report p
         |> Report.tryRaise
 
-    // Overload for ease-of-use from C#
-#if !FABLE_COMPILER
-    [<CompiledName("Check")>]
-#endif
     let checkBool (g : Property<bool>) : unit =
         bind g ofBool |> check
 
-    // Overload for ease-of-use from C#
-#if !FABLE_COMPILER
-    [<CompiledName("Check")>]
-#endif
     let checkBool' (n : int<tests>) (g : Property<bool>) : unit =
         bind g ofBool |> check' n
 
     /// Converts a possibly-throwing function to
     /// a property by treating "no exception" as success.
-    let internal fromThrowing (f : 'a -> unit) (x : 'a) : Property<unit> =
```

Comment:
Why the change from `internal` to `public` scope?

Reply:
I figured this is useful functionality, it can be changed back if that's not the case.

Reply:
I don't care either way.  Just wanted to draw some attention to this change.

Reply:
I've renamed it to `ofThrowing`, to match the existing naming.